MA-Bylaws.md

```diff
@@ -109,21 +96,25 @@ Between meetings the Board (see § 16 and § 17) is MA's point of contact.
 The Board will follow the applicable regulations and decisions made by
 the Assembly.
 
-MA's Board consists of:
+MA's Board consists of a minimum of four and maximum of nineteen members (and ten alternate members):
```

Comment:
I am not sure about the upper threshold: Does this mean that in case of many MAPs not all of their leaders are abrle to become Board member.

Reply:
I think this was a legal requirement to set an upper bound. If we once have too many MAPs, we can increase this.